test/arcade/patch/agent/process/PatchProcessMetabolismCARTTest.java

```diff
@@ -0,0 +1,263 @@
+        when(mockCell.getActivationStatus()).thenReturn(true);
+
+        metabolism.stepProcess(random, sim);
+
+        assertTrue(metabolism.intAmts[PatchProcessMetabolismCART.GLUCOSE] <= Double.MAX_VALUE);
+    }
+
+    @Test
+    public void stepProcess_withNegativeGlucoseConcentration_staysPositive() {
+        // mock inflammation process
+        PatchProcessInflammation inflammation = new InflammationMock(mockCell);
+        when(mockCell.getProcess(any())).thenReturn(inflammation);
+        when(mockCell.getActivationStatus()).thenReturn(true);
+
+        metabolism.stepProcess(random, sim);
+
+        assertTrue(metabolism.intAmts[PatchProcessMetabolismCART.GLUCOSE] >= 0);
+    }
+
+    @Test
+    public void stepProcess_withZeroOxygenConcentration_producesNoOxygen() {
+        // mock inflammation process
+        PatchProcessInflammation inflammation = new InflammationMock(mockCell);
+        when(mockCell.getProcess(any())).thenReturn(inflammation);
+        when(mockCell.getActivationStatus()).thenReturn(true);
+
+        metabolism.extAmts[PatchProcessMetabolismCART.OXYGEN] = 0.0;
+
+        metabolism.stepProcess(random, sim);
+
+        assertEquals(0.0, metabolism.extAmts[PatchProcessMetabolismCART.OXYGEN]);
+    }
+
+    @Test
+    public void stepProcess_withMaxOxygenConcentration_doesNotOverflow() {
+        // mock inflammation process
+        PatchProcessInflammation inflammation = new InflammationMock(mockCell);
+        when(mockCell.getProcess(any())).thenReturn(inflammation);
+        when(mockCell.getActivationStatus()).thenReturn(true);
+
+        metabolism.extAmts[PatchProcessMetabolismCART.OXYGEN] = Double.MAX_VALUE;
+
+        metabolism.stepProcess(random, sim);
+
+        assertTrue(metabolism.extAmts[PatchProcessMetabolismCART.OXYGEN] <= Double.MAX_VALUE);
+    }
+
+    @Test
+    public void step_whenActivated_producesMoreGlucose()
+            throws IllegalAccessException, NoSuchFieldException {
+        metabolism.extAmts[PatchProcessMetabolismCART.GLUCOSE] = 10000.0;
+        // mock inflammation process
+        PatchProcessInflammation inflammation = new InflammationMock(mockCell);
+        when(mockCell.getProcess(any())).thenReturn(inflammation);
+        metabolism.stepProcess(random, sim);
+        double inactiveGlucose = metabolism.intAmts[PatchProcessMetabolismCART.GLUCOSE];
+        double inactivePyruvate = metabolism.intAmts[PatchProcessMetabolismCART.PYRUVATE];
+        Field activeTicker = PatchProcessInflammation.class.getDeclaredField("activeTicker");
+        activeTicker.setAccessible(true);
+        activeTicker.set(inflammation, 1);
+        when(mockCell.getActivationStatus()).thenReturn(true);
+        metabolism.stepProcess(random, sim);
+        double activatedGlucose = metabolism.intAmts[PatchProcessMetabolismCART.GLUCOSE];
+        double activatedPyruvate = metabolism.intAmts[PatchProcessMetabolismCART.PYRUVATE];
+
+        assertTrue(activatedGlucose > inactiveGlucose);
```

Comment:
I would make this two tests, and calculate what the values should be in the activated state vs. inactive state.